resources/timerbox.ts

```diff
@@ -416,6 +431,12 @@ export default class TimerBox extends HTMLElement {
   advance(): void {
     const elapsedSec = (new Date().getTime() - this._start) / 1000;
     if (elapsedSec >= this._duration) {
+      // We need to check for this._duration > 0 here, as for undocumented reason the
+      // duration of a timerbox is always set to zero before it is set to the
+      // actual duration. As a result this would otherwise trigger a sound each time
+      // the ability is activated.
```

Comment:
I couldn't find any documentation or reason in a commit message why the duration is always set to zero before it is immediately after set to the correct value. Removing the assignment to zero didn't show any issues in my short tests. Does anyone remember what issues this solves?

Reply:
Ah sorry, these web components have existed foreverrrrrrr and are a bit dusty.

I think this would be worth tracking down and fixing separately.  My memory here is that some(?) of the web components in `resources/` needed to have some(?) attributes set to zero to properly reset them, but I don't remember why anymore.  It would be nice to not have to do this in general.